packages/rewriter/lib/threads/worker.ts

```diff
@@ -58,7 +58,7 @@ parentPort!.on('message', (req: RewriteRequest) => {
   }
 
   try {
-    const output = _getOutput()
+    const output = await _getOutput()
```

Comment:
@panzarino i thought this wasn't working like we expected, did you figure it out?

Reply:
It was working, turns out the issue I was seeing was from somewhere else and it was completely unrelated. I also tested the async functions specifically and it was working fine.